sherpa/fit.py

```diff
@@ -1128,6 +1327,56 @@ def simulfit(self, *others):
 
```

Comment:
If I understand this correctly `est_error()` calculates the confidence bounds for the model parameters. Suggest to change to

'Calculate the low and high confidence bounds for one or more of the thawed parameters in the fit.'